types/nightwatch/index.d.ts

```diff
@@ -4783,16 +4850,17 @@ export interface ElementCommands {
      *    });
      * });
      *
+     * @see https://nightwatchjs.org/api/getShadowRoot.html
      */
     getShadowRoot(
         selector: Definition | WebElement | By,
         callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<Element>) => void,
-    ): this;
+    ): Awaitable<this, Element>;
     getShadowRoot(
         using: LocateStrategy,
         selector: Definition | WebElement | By,
         callback?: (this: NightwatchAPI, result: NightwatchCallbackResult<Element>) => void,
-    ): this;
+    ): Awaitable<this, Element>;
```

Comment:
`Element` --> `Element | null`
It returns `null` if there's no shadow root to the element.